frontend/packages/dev-console/src/components/pipelines-lists/PipelinesListsPage.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { match as Rmatch } from 'react-router-dom';
+import MultiTabListPage from '../multi-tab-list/MultiTabListPage';
+import { referenceForModel, K8sKind } from '@console/internal/module/k8s';
+import {
+  PipelineModel,
+  PipelineResourceModel,
+  ConditionModel,
+  PipelineRunModel,
+} from '../../models';
+import { Page, Firehose } from '@console/internal/components/utils';
+import { filters } from '../pipelines/list-page/PipelineAugmentRuns';
+import PipelineAugmentRunsWrapper from '../pipelines/list-page/PipelineAugmentRunsWrapper';
+import { TechPreviewBadge } from '@console/shared';
+import PipelineRunsResourceList from '../pipelineruns/PipelineRunsResourceList';
+import { DefaultPage } from '@console/internal/components/default-resource';
+import PipelineResourcesListPage from '../pipeline-resources/list-page/PipelineResourcesListPage';
+
+interface PipelinesListPageProps {
+  match: Rmatch<any>;
+}
+
+export const PipelinesResourceList: React.FC<PipelinesListPageProps> = ({
+  match: {
+    params: { ns: namespace },
+  },
+}) => {
+  const resources = [
+    {
+      isList: true,
+      kind: referenceForModel(PipelineModel),
+      namespace,
+      prop: PipelineModel.id,
+      filters: { ...filters },
+    },
+  ];
+  return (
+    <div className="co-m-pane__body">
+      <Firehose resources={resources}>
```

Comment:
Mind using the hook instead of Firehose? I think we can eliminate both usages of Firehose that wrap `PipelineAugmentRunsWrapper` if you just use the hook inside that component by doing that.

Reply:
sure will use the hook

Reply:
@andrewballantyne I think we should have a separate task or story for refactoring `PipelineAugmentRunsWrapper` since `PipelineAugmentRuns` also consumes its props and the data returned by the hook needs to be modified in order to use it in  `PipelineAugmentRunsWrapper`. And the story can cover refactoring both `PipelineAugmentRunsWrapper` and  `PipelineAugmentRuns` and files which uses them.

Reply:
Makes sense. Thanks for doing the investigation. We can bump that.